Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM vvoronin/ubuntu-tools
```

Comment:
I actually have another image build based on perl:5.16.3 (https://hub.docker.com/_/perl/) , but the size of the image is about 500 MB. Check https://hub.docker.com/r/soleo/duckpan/tags/ . @moollaza do you think the size of the image is going to be an issue for other developers?